src/Cell.cpp

```diff
@@ -388,6 +506,88 @@ void Cell::setFill(Universe* fill) {
 
 
 /**
+ * @brief Set the Cell's rotation angles about the x, y and z axes.
+ * @details This method is a helper function to allow OpenMOC users to assign
+ *          the Cell's rotation angles in Python. A user must initialize a
+ *          length 3 NumPy array as input to this function. This function then
+ *          stores the data values in the NumPy array in the Cell's rotation
+ *          array. An example of how this function might be called in Python
+ *          is as follows:
+ *
+ * @code
+ *          rotation = numpy.array([0., 0., 90.])
+ *          cell = openmoc.Cell()
+ *          cell.setRotation(rotation)
+ * @endcode
+ *
+ * @param rotation the array of rotation angles
+ * @param num_axes the number of axes (this must always be 3)
+ */
+void Cell::setRotation(double* rotation, int num_axes) {
+
+  if (num_axes != 3)
+    log_printf(ERROR, "Unable to set rotation with %d axes for Cell %d. "
+               "The rotation array should be length 3.", num_axes, _id);
+
+  for (int i=0; i < 3; i++)
+    _rotation[i] = rotation[i];
+
+  /* Compute rotation angles in x,y,z directions */
+  double phi = -_rotation[0] * M_PI / 180.;
+  double theta = -_rotation[1] * M_PI / 180.;
+  double psi = -_rotation[2] * M_PI / 180.;
+
+  /* Calculate rotation matrix based on angles given */
+  /* Indexed by (y,x) since the universe array is indexed by (z,y,z) */
```

Comment:
I'm having a hard time understanding this comment. Why only two coordinates in the first parentheses and three in the second parentheses? Also, what do you think about making the matrix a 2D array and stating whether it is row-major or column-major?